src/utils/getActiveTab.ts

```diff
@@ -0,0 +1,22 @@
+import { Tab } from '@/interfaces/task.type';
+
+export const getActiveTab = (section?: string): Tab => {
+    switch (section) {
+        case 'assigned':
```

Comment:
Don't we have similar mappings already? If yes, then can we reuse it?

Reply:
I did not find any mapping to match this